keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -839,7 +863,11 @@ conversion tests if API types are being modified.
 This section must be completed when targeting beta to a release.
 -->
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
+Feature does not have a direct impact on rollout/rollback. 
+However, the faulty behavior of a feature can lead to the incorrect functioning of kube-controller-manager. 
```

Comment:
Let's maybe clarify it a bit:

... can lead to the incorrect functioning of components using the feature. For Beta we're planning to enable it for kube-controller-manager.